README.md

```diff
@@ -123,29 +123,29 @@
   - [mydumper](tools/mydumper.md)
   - [Loader](tools/loader.md)
   + Data Migration
-    - [Overview](/tools/dm/overview.md)
-    - [Restrictions](/tools/dm/overview.md#usage-restrictions)
-    - [Deploy](/tools/dm/practice.md)
+    - [Overview](tools/dm/overview.md)
+    - [Restrictions](tools/dm/overview.md#usage-restrictions)
+    - [Deploy](tools/dm/practice.md)
     + Features
-      - [Table Routing](/tools/dm/data-synchronization-features.md#table-routing)
-      - [Black and White Lists](/tools/dm/data-synchronization-features.md#black-and-white-table-lists)
-      - [Binlog Event Filtering](/tools/dm/data-synchronization-features.md#binlog-event-filtering)
-      - [Column Mapping](/tools/dm/data-synchronization-features.md#column-mapping)
-      - [Synchronization Delay Monitoring](/tools/dm/data-synchronization-features.md#synchronization-delay-monitoring)
+      - [Table Routing](tools/dm/data-synchronization-features.md#table-routing)
+      - [Black and White Lists](tools/dm/data-synchronization-features.md#black-and-white-table-lists)
+      - [Binlog Event Filtering](tools/dm/data-synchronization-features.md#binlog-event-filtering)
+      - [Column Mapping](tools/dm/data-synchronization-features.md#column-mapping)
+      - [Synchronization Delay Monitoring](tools/dm/data-synchronization-features.md#synchronization-delay-monitoring)
       + Sharding Support
-        - [Introduction](/tools/dm/sharding-solution.md)
-        - [Restrictions](/tools/dm/sharding-solution.md#sharding-ddl-usage-restrictions)
-        - [Troubleshoot](/tools/dm/troubleshooting-sharding-ddl-locks.md)
+        - [Introduction](tools/dm/shard-merge.md)
+        - [Restrictions](tools/dm/shard-merge.md#restrictions)
+        - [Handle Sharding DDL Locks manually](tools/dm/manually-handling-sharding-ddl-locks.md)
```

Comment:
link to file added in #876. so we need to merge #876 as soon as this PR merged?

Reply:
Yes.